saleor/payment/utils.py

```diff
@@ -33,26 +33,33 @@ def create_payment_information(
     customer_id: str = None,
     store_source: bool = False,
     additional_data: Optional[dict] = None,
+    partial: bool = False,
 ) -> PaymentData:
     """Extract order information along with payment details.
 
     Returns information required to process payment and additional
     billing/shipping addresses for optional fraud-prevention mechanisms.
     """
     checkout = payment.checkout
+    checkout_token = None
     if checkout:
         billing = checkout.billing_address
         shipping = checkout.shipping_address
         email = checkout.get_customer_email()
         user_id = checkout.user_id
+        checkout_token = str(checkout.token)
     elif payment.order:
         billing = payment.order.billing_address
         shipping = payment.order.shipping_address
         email = payment.order.user_email
         user_id = payment.order.user_id
+        checkout_token = str(payment.order.checkout_token)
     else:
         billing, shipping, email, user_id = None, None, payment.billing_email, None
 
+    if not checkout_token:  # Either not empty uuid or None
```

Comment:
Object.checkout_token can has empty string, bu it not valid uuid.

Reply:
It looks like it can happen only for `payment.order` case. (and it can happen only for DraftOrder which currently doesn't support payments). Instead of adding special if for this case, can't we do something like:
```
elif  payment.order:
    ...
    checkout_token = payment.order.checkout_token or None
```

Reply:
OK